CHANGELOG.md

```diff
@@ -8,6 +8,20 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+This release adds experimental logs support. Set `OTEL_LOGS_EXPORTER=otlp` to
+enable logs support. However, take notice that, as of now, the OpenTelemetry Go
+Logs API is not stable.
+
+### Added
+
+- Added logs support and the `OTEL_LOGS_EXPORTER` environment variable.
+- `OTEL_LOGS_EXPORTER` accepts:
+  - `none` - logs disabled.
+  - `otlp` - OTLP gRPC exporter.
+- Currently, `OTEL_LOGS_EXPORTER` defaults to `none` as the OpenTelemetry Go
+  Logs API and SDK are not stable yet.
+- Set `OTEL_LOGS_EXPORTER=otlp` to enable logs support.
```

Comment:
This should be one entry (not 4).

Reply:
https://github.com/signalfx/splunk-otel-go/pull/3673/commits/d7ecd5b5f845ba2a583bc546202dd2358527253b